### ISSUE TITLE
Default window size

### ISSUE THREAD

**Issue by shawn1thompson** [gpodder/gpodder#578]
Is there a way to configure the default window size in Linux? It doesn't save the size I change it to once I close the app. The download window is even worse; somehow it defaults to full screen now.

Thank you.

**Comment by elelay** [comment]
gPodder should remember the size (and location?) of the window from last run (see `$GPODDER_HOME/Settings.json` : the `ui > gtk > state > main_window` section).
What window manager do you use? I'm using i3 and it doesn't work when the window is tiled, but it works when I make it float.

**Comment by shawn1thompson** [comment]
Thanks for responding. I use the Pantheon DE (elementary OS Juno).

**Comment by elelay** [comment]
Don't you have  a `main_window` section after `episode_selector`?
```
        "main_window": {
          "episode_list_size": 244,
          "height": 967,
          "maximized": false,
          "paned_position": 391,
          "width": 1300,
          "x": 560,
          "y": 197
        }
```

**Comment by shawn1thompson** [comment]
Yes. I changed the numbers there and that corrected the issue. Up to you if you want to investigate further. I'll provide any info you  might want.

**Comment by elelay** [comment]
Good to know that it's fixed for you! Is it possible that you once changed screen resolution or plugged a bigger screen, causing the episode_selector size to be bigger than your current desktop?

**Comment by shawn1thompson** [comment]
Not that I can recall. I do recall it happening suddenly,  but don't remember what precipitated it unfortunately.

**Comment by shawn1thompson** [comment]
I'm  not sure it is fixed. The download window just popped up with the wrong size although not maximized like it was before. Another thing I neglected to mention earlier is that when the download window comes up with the wrong size, the clicks pass through to the underlying window. For example, if I click the download button, it affects the underlying window instead of downloading. The only way I've found around this is to use the Enter key to activate the download button.
